sample/HttpTrigger-CSharp-Legacy/function.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bindings": [
```

Comment:
Might consider moving this legacy function out of samples which for the most part we want to remain a set of clean samples to follow, and instead do this in an integration tests function?

Also, I don't see this in the solution explorer. Is it not added to the sln or something?

Reply:
Removed